tests/LSP17ContractExtension/LSP17Extendable.behaviour.ts

```diff
@@ -65,7 +68,8 @@ export const shouldBehaveLikeLSP17 = (
     revertStringFunctionExtensionHandlerKey,
     revertCustomFunctionExtensionHandlerKey,
     emitEventFunctionExtensionHandlerKey,
-    onERC721ReceivedFunctionExtensionHandlerKey;
+    onERC721ReceivedFunctionExtensionHandlerKey,
+    supportsInterfaceFunctionExtensionHandlerKey;
 
   beforeEach(async () => {
```

Comment:
Here `before` should enough too.

```suggestion
  before(async () => {
```